gems/sorbet-runtime/lib/types/configuration.rb

```diff
@@ -130,7 +130,7 @@ def self.default_checked_level=(default_checked_level)
   #
   # Parameters passed to value.call:
   #
-  # @param [TypeError] error TypeError that was raised
+  #  @param [TypeError] error TypeError that was raised
```

Comment:
The extra space seems to be enough to prevent `yard` from trying to attach the tag to a method param (but the resulting formatting is terrible)